src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefConditionalOperatorTests.cs

```diff
@@ -918,6 +918,142 @@ .maxstack  1
         }
 
         [Fact]
+        public void TestRefConditionalHomelessBranches()
+        {
+            var source = @"
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var o = new C1();
+            var o1 = new C1();
+
+            (args != null ? ref o.field : ref o1.field).ToString();
```

Comment:
Consider similar test with a ref readonly extension method instead of `ToString`. Then we expect no copy for the invocation (but also no mutation...).